pephubclient/helpers.py

```diff
@@ -0,0 +1,61 @@
+import json
+from typing import NoReturn, Optional
+
+import requests
+
+from pephubclient.exceptions import ResponseError
+
+
+class RequestManager:
```

Comment:
Another example of a class that might be able to be two functions. However, I think a requests manager could be useful. I've used this paradigm before. [See here](https://requests.readthedocs.io/en/latest/user/advanced/).

We can create a session in the object to handle things globally. I think [this](https://github.com/johnwmillr/LyricsGenius/blob/master/lyricsgenius/api/base.py#L9) is a good example of such an object.